tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -912,6 +912,7 @@
         '/api/webhooks',
         '/api/webhooks/:id',
         '/api/webhooks/:id',
+        '/api/webhooks/:id/test',
```

Comment:
I don't think this endpoint has anything to do with that BZ.

Reply:
Ah, true, I believe this is a feature, introduced with https://github.com/theforeman/foreman_webhooks/pull/57 (don't have the matching bz at hand right now)